README.md

```diff
@@ -226,34 +227,32 @@ git clone --recursive https://github.com/nesbox/TIC-80 && cd TIC-80/build
 cmake -DBUILD_SDLGPU=On -DBUILD_WITH_ALL=On .. && cmake --build . --parallel
 ```
 
-Install with [Install instructions](#install-instructions)
-
+Install with [Install Instructions](#install-instructions)
 
 ### Arch
-
 run the following commands in the Terminal
 ```
 sudo pacman -S cmake ruby mesa libglvnd glu
 git clone --recursive https://github.com/nesbox/TIC-80 && cd TIC-80/build
 cmake -DBUILD_WITH_ALL=On ..
-make -j4
```

Comment:
I think we need this command to build binaries, or we can use `cmake --build . --parallel` instead